modules/accredited_representative_portal/app/services/accredited_representative_portal/power_of_attorney_request_service/accept.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  module PowerOfAttorneyRequestService
+    class Accept
+      attr_reader :poa_request, :creator, :reason, :resolution
+      attr_accessor :form_submission
+
+      def initialize(poa_request, creator, reason)
+        @poa_request = poa_request
+        @creator = creator
+        @reason = reason
+      end
+
+      def call
+        ApplicationRecord.transaction do
+          create_resolution!
+          create_form_submission!
+        end
+
+        service = BenefitsClaims::Service.new(poa_request.claimant.icn)
+        response = service.submit2122(attributes)
+
+        form_submission.update(
+          service_id: response.body.dig('data', 'id'),
+          service_response: response.body.to_json
+        )
+        form_submission
+      # TODO: call PowerOfAttorneyFormSubmissionJob.perform_async(poa_form_submission)
+      # Transient 5xx errors: delete objects created, re-raise error
+      rescue *transient_error_types => e
```

Comment:
If this sticks around I would actually make it a constant. Fine to do a calculation of another constant like that (and freeze it potentially): `TRANSIENT_ERROR_TYPES`.